tools/FETCH-PROTOC.sh

```diff
@@ -27,12 +27,12 @@ case "$(arch)" in
   ;;
 esac
 
-export VERSION="3.19.3"
+. tools/PROTOC-VERSION.sh
```

Comment:
Is "source" equivalent to "." ? We use that elsewhere in the project (maybe everywhere else?)

Reply:
Thanks - If you don't mind, let's go for consistency (short term) and do the replacement as a unit.

Reply:
I could do that, but unfortunately, that will break it... it's the reason why I used `.` in the first place.

Reply:
Well, actually, now that I think about it... it might've been because I used `sh xxxx` to execute something. I'd have to go back a re-test to see what may or may not break...

Reply:
I've opened the [issue](https://github.com/apigee/registry/issues/545). If you'd prefer, I can remove this change from this issue and can fix it in the other. Or, I can remove this entirely and you can close the issue if you just don't like it.